java/arcs/core/util/Dispatchers.kt

```diff
@@ -0,0 +1,16 @@
+package arcs.core.util
+
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+
+object Dispatchers {
```

Comment:
Add KDoc something like "Centralized dispatcher references for use across Arcs to allow Arcs embeddings to customize with their own dispatchers as needed."

I wonder whether we should do this differently later injection-wise, since static initializer order of execution is probably not guaranteed, so if for example there is other Arcs Kt code that does something with dispatchers in `JvmStatic` it could grab these original Kt dispatchers before the embedding client code replaces them with others.

We could add to the KDoc to make clear that Arcs code should avoid doing this for now.

Reply:
We should definitely remove this and replace it with injectable dispatchers soon. Global variables are disasters for testing. 
This is only a fats-path to quickly replace all references with customized dispatchers.

Reply:
For internal usage, we have AppInitilizer to guarantee these variables are set at app start at per-thread granularity so I think the ordering issue could be solved.